pyface/ui/wx/preference/preference_dialog.py

```diff
@@ -134,7 +134,7 @@ def _create_tree(self, parent):
             content_provider=DefaultTreeContentProvider(),
         )
 
-        tree_viewer.on_trait_change(self._on_selection_changed, "selection")
+        tree_viewer.observe(self._on_selection_changed, "selection")
```

Comment:
I can't comment on the line itself, but this file has `from pyface.ui.wx.split_dialog import SplitDialog` but there is no `split_dialog` module in `pyface/ui/wx` ...